docs/bridge/architecture.mdx

```diff
@@ -3,10 +3,55 @@ title: Architecture
 sidebar_position: 1
 ---
 
+## Overview
 
-Here is a high level image of all the components of the bridge. 
+The Calimero Bridge handles the transfer of various assets, including fungible and non-fungible tokens, between NEAR public networks (Testnet or Mainnet) and Calimero private shard. It also facilitates cross-shard contract calls, enabling interactions between public and private smart contracts.
 
-<img src={require('@site/static/img/bridge_components.jpg').default} />
+## Workflow Diagram
 
+Below is a high-level overview of the bridge's components:
 
-Read more about each component in subsequent sections.
+![Workflow Diagram](https://github.com/calimero-is-near/docs/assets/39309699/c63274ad-6873-4b86-bc4b-5c10bdd4e28d)
+
+## Key Components
+
+The architecture of the NEAR to Calimero bridge includes the following key components:
+
+### Light Client Contract
+
+The Light Client Contract validates and accepts block headers that meet specific validation criteria, ensuring the integrity and security of the bridge's operations.
+
+### Prover Contract
+
+The Prover Contract provides verifiable proofs of events or actions that occurred on a specific chain, enhancing transparency and trust.
```

Comment:
This is not a precise explanation. The Prover contract proves that a transaction occurred on the opposite chain (for example on the Near side if this Prover contract is on the Calimero side). Prover contract asks light client contract for the Merkle tree root for the specific block height, and he uses this information for the verification of the proof